### PR TITLE
Mf 6 navbar

### PR DESCRIPTION
**Pull request by BrandonAlexanderLopez**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

* **What is the current behavior?What does the code do before you made the new changes?** 

* **What is the new behavior?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

* **Other information**:

### CODE REVIEW

**Comment by BrandonAlexanderLopez** [comment]
_What kind of change does this PR introduce? (Bug fix, new feature...)_
**The navbar has been added to the application.**

_What is the current behavior? What does the code do before you made the new changes?_
**You may now select options in the menu. Previous to these changes there was no existing navbar. 
Some pages aren't finished yet meaning selecting certain options in the menu returns no function.** 

_What is the new behavior?_
**The new behavior of our app is that it allows users to make selections in the newly available navbar.**

_Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR? Run a new command?)_
**As is nothing is BROKEN, but the navbar still has some work to be done. It is not fully functional yet, however there is currently no breaks.**

**Review by Sakamoto448 — DISMISSED** [review]
Tried it out, Nothing breaks and all components work. Looks good to me, nice job!

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                  </DropdownItem>
+                  <DropdownItem>
+                    Option 2
+                  </DropdownItem>
+                  <DropdownItem divider />
+                  <DropdownItem>
+                    Reset
+                  </DropdownItem>
+                </DropdownMenu>
+              </UncontrolledDropdown>
```
Comment (Codubee) [inline src/components/Navigation.js]: I think a lot of these links are not needed for now. What do you think about removing them?

**Review by Codubee — CHANGES_REQUESTED** [review]
Left a comment for discussion

File: src/components/Navigation.js (hunk 1 above)
Comment (BrandonAlexanderLopez) [reply src/components/Navigation.js]: Agreed, will do.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**